api/HTMLUnknownElement.json

```diff
@@ -23,10 +23,10 @@
             "version_added": true
           },
           "opera": {
-            "version_added": true
+            "version_added": "≤12.1"
```

Comment:
`HTMLUnknownElement` is a special interface which we test in a special way. Confirmed with http://mdn-bcd-collector.appspot.com/tests/api/HTMLUnknownElement on Opera 10.6 that it returns true. The test is prone to false negatives because it verifies a bunch of things, but not false positives. I'm not sure, but I think `HTMLUnknownElement` goes back to the dawn of the browser era.